packages/twenty-server/src/engine/api/graphql/workspace-query-runner/factories/query-runner-args.factory.ts

```diff
@@ -210,30 +206,12 @@ export class QueryRunnerArgsFactory {
             'Rich text is not supported, please use RICH_TEXT_V2 instead',
           );
         case FieldMetadataType.RICH_TEXT_V2: {
-          const richTextV2Value = richTextV2ValueSchema.parse(value);
-
-          const serverBlockNoteEditor = ServerBlockNoteEditor.create();
-
-          const convertedMarkdown = richTextV2Value.blocknote
-            ? await serverBlockNoteEditor.blocksToMarkdownLossy(
-                JSON.parse(richTextV2Value.blocknote),
-              )
-            : null;
-
-          const convertedBlocknote = richTextV2Value.markdown
-            ? JSON.stringify(
-                await serverBlockNoteEditor.tryParseMarkdownToBlocks(
-                  richTextV2Value.markdown,
-                ),
-              )
-            : null;
-
-          const valueInBothFormats: RichTextV2Metadata = {
-            markdown: richTextV2Value.markdown || convertedMarkdown,
-            blocknote: richTextV2Value.blocknote || convertedBlocknote,
-          };
-
-          return [key, valueInBothFormats];
+          const overriddenValue = await overrideFieldValue(
```

Comment:
it feels wrong to have the RICH_TEXT_V2 case in the overrideFieldValue but not NUMBER or POSITION

Looks like what we want is:
- a nestJS service (independent from graphql and nestJS) that can take an record input and transform it based on fieldMetadata and the input => which is more or less taking this query-runner-args factory and remove the graphql coupling ==> I understand the idea of having a shared util but I think we should go one step further and have a shared service (long term vision, this should be part of the ORM layer IMO)


Naming proposal
- const processedRecordInput = this.recordInputTransformer.process({ recordInput, objectMetadata }}